modules/stuffed/tomcat-resource.json

```diff
@@ -73,9 +73,13 @@
     {"name":"org.apache.tomcat.websocket.server.LocalStrings"}
   ],
   "resources":[
-    {"pattern":".*/mbeans-descriptors.xml$"},
-    {"pattern":".*/*.properties$"},
-    {"pattern":".*/*.dtd$"},
-    {"pattern":".*/*.xsd$"}
```

Comment:
Is a pattern no longer (or was never?) necessary for `.xsd`?

Reply:
They are now included via `{"pattern":"^jakarta/servlet/resources/.*"}`.